doc/source/notebooks/advanced/known-noise.ipynb

```diff
@@ -0,0 +1,331 @@
+      "text/plain": [
+       "<Figure size 864x432 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(12, 6))\n",
+    "plt.errorbar(X, Y, yerr=2*(np.sqrt(Sigma)), marker='x', lw=0, elinewidth=1., color='C1')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Make a Y matrix that includes the variances\n",
+    "Since GPflow models are based on generalised GP models, we need to coerce out modelling assumptions into that form. All that's needed is to stack the observations with their respective  variances.\n"
```

Comment:
I think it would be nice to write the modelling assumptions explicitly here. As far as I understand the notebook, we have a multioutput GP with the first output being the mean function and the second output is the variance of the Gaussian likelihood?